sdk/keyvault/azure-security-keyvault-secrets/pom.xml

```diff
@@ -79,14 +79,14 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-test</artifactId>
-      <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
+      <version>1.3.1</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
       <scope>test</scope>
     </dependency>
 
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.0.7</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
```

Comment:
Ideally, these changes should not have been a part of this PR.
cc: @JimSuplizio

Reply:
So we keep changes like these when merging from master and only bump versions we control (e.g. KeyVault), right?

Reply:
yes, sorry missed this was the hotfix branch.